projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -126,6 +126,27 @@ describe('AuthService', () => {
         done();
       });
     });
+
+    it('should return null when logged out', (done) => {
+      const user = {
+        name: 'Test User',
+      };
+
+      (auth0Client.isAuthenticated as jasmine.Spy).and.resolveTo(true);
+      (auth0Client.getUser as jasmine.Spy).and.resolveTo(user);
+
+      service.user$.pipe(bufferCount(2)).subscribe((values) => {
```

Comment:
In this test I am using `bufferCount` to be able to run a test on the last 2 emitted items, to verify it changes from a user to null.